flexmeasures/app.py

```diff
@@ -143,7 +143,7 @@ def get_reporters():
         )
         return app.data_generators["reporter"]
 
-    setattr(app, "reporters", property(get_reporters))
+    setattr(app, "reporters", get_reporters())
```

Comment:
Shouldn't we be repeating this pattern for schedulers, too?

Reply:
Only reporters are treated this way, probably because they existed on app.reporters already (for a few weeks).

Or tell me the spot that I overlooked.

Reply:
In lines 130-134 we store reporters and schedulers under `app.data_generators`. This is our new way.

In lines 137-138 we still set up `app.reporters` and `app.schedulers`. This is the old way, which we no longer want. Imo these two lines should not be there anymore. But we want to nicely deprecate them, by preserving backwards compatibility while logging a warning.

In lines 140-146 we attempt to do just that, by nicely deprecating `app.reporters` (which apparently had a bug). With my previous comment I'm raising the issue that we failed to implement the nicely deprecating of `app.schedulers`.

Hope this clears it up?

Reply:
Yes. I'm questioning if this bugfix PR needs to do that, as it's blocking the release.

I'm no expert on that topic, also